packages/documentation-site/patternfly-docs/content/design-guidelines/usage-and-behavior/usage-and-behavior.md

```diff
@@ -3,119 +3,138 @@ id: Usage and behavior
 
+- Tooltips are shown when users hover over an element. 
 
-|   Use case              |[Tooltip](/components/tooltip) | [Popover](/components/popover) | [Hint](/components/hint) |
-|------------------------ | :---: | :-------: | :---------------: |
-|You want to provide a short (no more than 1-2 lines) explanation of new or unfamiliar UI elements as a simple text string that is only shown “on-demand.”|✔|⛔ | ⛔ |
-|You want to include formatted text and/or interactive elements in your message body.|⛔|✔|✔|
-|You want the information to persist until the user dismisses it.|⛔|✔|✔|
-|You want the information to be announced by a screen reader whenever the user tabs to an element.|✔|✔ [1]|✔|
-|You want to present additional information that might not be necessary or relevant to all users.|⛔|✔|✔[2]|
+- Popovers can be shown on hover or click, can contain any HTML content, and are persistent. 
 
-**[1]** By default, popovers are only triggered when the user clicks on an element and therefore will not be read by a screen reader when tabbing through an interface. If a popover is triggered on hover (optional behavior), it will behave like a tooltip and its content will be displayed whenever the keyboard user tabs to the trigger element.
+- Hints are static elements that appear inline with other content on a page.
 
-**[2]** Hint could be used to convey information to advanced users (“pro-tips,” for example). However because a hint adds static content directly to the page, consider whether it’s important for this information to be shown at all times.
+Your use case dictates which of these components you should use:
+
+| **Use case** | **Recommended component** |
+|------------------------ | -----------|
+| You want to provide a short (no more than 1-2 lines) explanation of new or unfamiliar UI elements as a simple text string that is only shown “on-demand.” |Tooltip|
+| You want to include formatted text and/or interactive elements in your message body. | Hint or popover |
+| You want the information to persist until the user dismisses it. | Hint or popover |
+|You want the information to be announced by a screen reader whenever the user tabs to an element.| Hint, popover, or tooltip <br /><br /> **Note:** By default, popovers are only triggered when the user clicks on an element. Therefore screen readers will not read popover text when tabbing through an interface. If a popover is triggered on hover (optional behavior), it will behave like a tooltip and its content will be announced whenever a keyboard user tabs to the trigger element. |
+|You want to present additional information that might not be necessary or relevant to all users.| Hint or popover <br /><br /> **Note:** Hint can be used to convey information to advanced users (such as “pro-tips”. However, since a hint adds static content to a page, you should consider whether it’s important for this information to be shown at all times.|
```

Comment:
```suggestion
|You want to present additional information that might not be necessary or relevant to all users.| Hint or popover <br /><br /> **Note:** Hints can be used to convey information to advanced users (such as “pro-tips”. However, since a hint adds static content to a page, you should consider whether it’s important for this information to be shown at all times.|
```